client/state/concierge/info/reducer.js

```diff
@@ -0,0 +1,22 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { combineReducers, createReducer, keyedReducer } from 'state/utils';
+import { CONCIERGE_INFO_UPDATE } from 'state/action-types';
+
+export const message = createReducer( null, {
+	[ CONCIERGE_INFO_UPDATE ]: ( state, action ) => action.info.message,
+} );
+
+export const timezone = createReducer( null, {
+	[ CONCIERGE_INFO_UPDATE ]: ( state, action ) => action.info.timezone,
+} );
+
+export const reducer = combineReducers( {
+	message,
+	timezone,
+} );
+
+export default keyedReducer( 'siteId', reducer );
```

Comment:
I'm not sure we need a keyed reducer on siteID. For now you can't switch site ID without altering the URL, so we'll only have one site's data in state at once.

Thinking ahead to when we might let customers select their site in a dropdown (if they have multiple business sites), I still don't think we'd want to key this info on site. If you fill out the form and then switch the site in the dropdown, we wouldn't want the other form info to change under you.

Reply:
Updated.